client/dashboard/task-list/index.js

```diff
@@ -256,6 +256,64 @@ class TaskDashboard extends Component {
 		);
 	}
 
+	onSkipStoreSetup = () => {
+		const completedTaskKeys = this.getTasks()
+			.filter( ( x ) => x.completed )
+			.map( ( x ) => x.key );
+
+		recordEvent( 'wcadmin_tasklist_skip', {
+			completed_tasks_count: completedTaskKeys.length,
+			completed_tasks: completedTaskKeys,
+			reason: 'skip',
+		} );
+
+		this.props.updateOptions( {
+			woocommerce_task_list_hidden: 'yes',
+		} );
+	};
+
+	onDoThisLater = () => {
+		const completedTaskKeys = this.getTasks()
+			.filter( ( x ) => x.completed )
+			.map( ( x ) => x.key );
+
+		recordEvent( 'wcadmin_tasklist_skip', {
+			completed_tasks_count: completedTaskKeys.length,
+			completed_tasks: completedTaskKeys,
+			reason: 'later',
+		} );
+
+		this.props.updateOptions( {
+			woocommerce_task_list_do_this_later: true,
+		} );
+	};
+
+	renderSkipActions() {
+		const { doThisLater } = this.props;
+
+		return (
+			<div className="skip-actions">
+				<Button
+					isLink={ true }
+					className="is-secondary"
+					onClick={ this.onSkipStoreSetup }
+				>
+					{ __( 'Skip store setup', 'woocommerce-admin' ) }
+				</Button>
+				{ ! doThisLater && ' | ' }
+				{ ! doThisLater && (
+					<Button
+						isLink={ true }
+						className="is-secondary"
+						onClick={ this.onDoThisLater }
+					>
+						{ __( "I'll do this later", 'woocommerce-admin' ) }
```

Comment:
After this button is clicked, the menu opens up but if you're not paying close attention it may seem like nothing happens since the page does not change.  I'm wondering if we want to show any kind of notice on click?  /cc @pmcpinto @jameskoster

Reply:
When you click that button the rest of the page does load in, but I agree that it isn't very obvious.